src/ui/screens/AgencyQR.tsx

```diff
@@ -280,7 +306,7 @@ export default class AgencyQR extends React.Component<QRProps, QRState> {
     render() {
         const {isConnectionReady, verifying} = this.state;
         return (
-            <div id={this.props.agentType} className="flex-block column">
+            <div id={CONSTANTS.verification_options[authIndex].id} className="flex-block column">
```

Comment:
`CONSTANTS.verification_options[authIndex].id` this is a long string that takes a bit to parse. Consider extracting this once and make it available with a more descriptive name.